backend/tests/api/get.ping.test.ts

```diff
@@ -0,0 +1,10 @@
+import { superdeno } from "superdeno/mod.ts";
+import app from '../import.app.symlink.ts';
+
+Deno.test("GET /api/ping that it returns returns PONG!", async () => {
+    await superdeno(app)
+      .get("/api/ping")
+      .expect("Content-Type", /^text/)
+      .expect("Pong!")
+      .expect(200);
+  });
```

Comment:
Seems like this file wasn't formatted using `trex run fmt`, as the newline is missing and doublequotes are used to import superdeno. Doesn't the fmt command work for our test files?

Reply:
@p-fruck Actually, idk 😅 Will check this

Reply:
@p-fruck Fixed in [b287575](https://github.com/Music-Bot-for-Jitsi/Jimmi/pull/95/commits/b287575d5e8f1faaac998842bdf037aa3c7d7cdb)

Reply:
Very nice :+1: